tests/run-fail/inception.rs

```diff
@@ -0,0 +1,32 @@
+// error-pattern:no mir for DefId
```

Comment:
Hilarious test. :)

But yeah, it would need crate metadata to contain MIR for *every* function. And later it would fall down on some FFI call, but it would be fun to see how far it can go.